datasets/reuters.py

```diff
@@ -64,9 +53,9 @@ def sort_key(ex):
         return len(ex.text)
 
     @classmethod
-    def splits(cls, path, train=os.path.join('Reuters-21578', 'data', 'reuters_train.tsv'),
-               validation=os.path.join('Reuters-21578', 'data', 'reuters_validation.tsv'),
-               test=os.path.join('Reuters-21578', 'data','reuters_test.tsv'), **kwargs):
+    def splits(cls, path, train=os.path.join('Reuters', 'train.tsv'),
+               validation=os.path.join('Reuters', 'dev.tsv'),
```

Comment:
Same here.

Reply:
It doesn't break previous code. Directory structure is different (in a way simpler) in Hedwig when compared to Castor.